src/Core/src/Handlers/Window/WindowHandler.Windows.cs

```diff
@@ -193,12 +193,24 @@ void OnWindowChanged(AppWindow sender, AppWindowChangedEventArgs args)
 
 			UpdateVirtualViewFrame(sender);
 		}
-
 		void UpdateVirtualViewFrame(AppWindow appWindow)
 		{
+			var hwnd = PlatformView.GetWindowHandle();
+			var bounds = hwnd.GetExtendedFrameBounds();
```

Comment:
Before calling GetExtendedFrameBounds(), confirm hwnd is not null or invalid to prevent runtime errors.

```
if (hwnd == IntPtr.Zero)
    return;
```